src/style.css

```
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+body {
+  font-size: 16px;
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto;
+}
+
+.search-form_big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-form_small {
+  width: 100%;
+  height: 42px;
+}
```

Do not repeat your self. Also do not hardcode height - use input padding instead
```suggestion
.search-form {
  display: flex;
  justify-content: center;
  margin: 20px auto;
}

```